.buildkite/pipeline.yml

```diff
@@ -158,6 +158,23 @@
       --test_tag_filters=-kubernetes,-jenkins_only,client_tests,-flaky
       --test_env=RAY_CLIENT_MODE=1 --test_env=RAY_PROFILING=1
       python/ray/tests/...
+- label: ":python: (Small & Large)(DEBUG)"
+  conditions: ["RAY_CI_PYTHON_AFFECTED"]
+  commands:
+    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT
+    - bazel test --config=ci $(./scripts/bazel_export_options)
+      --compilation_mode=dbg
+      --test_tag_filters=-kubernetes,-jenkins_only,-medium_size_python_tests_a_to_j,-medium_size_python_tests_k_to_z,-flaky,-post_wheel_build,-worker-container
```

Comment:
cc @ericl
Wonder if we should just start from object layer/basic tests? Maybe we can run

```
test_object_manager.py
test_object_spilling.py
test_object_spilling_2.py
test_actor.py
test_scheduling.py
test_array.py
test_distributd_sort.py
```
And we can run ASAN on the same set of tests (and we iteratively expand to other set of tests).

Reply:
that's a great idea.